File: Examples/PHP/Bookmarks/GetDocumentBookmarksChildren.php
//ExStart:
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\PDF\PdfApi;
use Aspose\PDF\AsposeApp;

class Bookmark {

	public $pdfApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->pdfApi = new PdfApi();
	}

	public function getDocumentBookmarksChildren() {
		// Upload file to Aspose Cloud Storage
		$fileName = "Sample-Bookmark.pdf";
		Utils::uploadFile($fileName);

		$result = $this->pdfApi->GetDocumentBookmarksChildren($name = $fileName, $bookmarkPath = "1", $storage = "", $folder = "");
		print_r ( $result );        
	}
}

$bookmark = new Bookmark();
$bookmark->getDocumentBookmarksChildren();

?>
//ExEnd:

File: Examples/DotNET/CSharp/Bookmarks/GetDocumentBookmarksChildren.cs
using System;
using Com.Aspose.PDF.Api;
using Com.Aspose.PDF.Model;
using Com.Aspose.Storage.Api;

namespace Bookmarks
{
    class GetDocumentBookmarksChildren
    {
        public static void Run()
        {
            // ExStart:1
            PdfApi pdfApi = new PdfApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);

            string fileName = "Sample-Bookmark.pdf";
            string bookmarkPath = null;
            string storage = null;
            string folder = null; 

            try
            {
                // Upload source file to aspose cloud storage
                storageApi.PutCreate(fileName, "", "", System.IO.File.ReadAllBytes(Common.GetDataDir() + fileName));

                // Invoke Aspose.PDF Cloud SDK API to get bookmarks children
                ResponseMessage apiResponse = pdfApi.GetDocumentBookmarksChildren(fileName, bookmarkPath, storage, folder);

                if (apiResponse != null && apiResponse.Status.Equals("OK"))
                {                   
                    Console.WriteLine("Get Bookmarks Children from a PDF,Done!");
                    Console.ReadKey();
                }
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);

            }
            // ExEnd:1
        }
    }
}
